uilib-docs/src/templates/component.js

```diff
@@ -138,6 +138,16 @@ export default class ComponentTemplate extends Component {
     return notes.map(this.renderNote);
   }
 
+  renderConfigurationLine(configuration) {
```

Comment:
See my previous comment. 
No need for another docs section. 

Also, I have a huge PR with lots of changes and refactor related to the new design. 
I rather don't have any conflicts there.

Reply:
We can wait with merging this.

Reply:
the big PR with the new design is merged. 
But I still wish to use a single field for general Component notes

Reply:
Is this the change I see in the docs? Nice ❗ 

I'll try to come up with a different solution.

Reply:
Maybe we can make our `notes` field super generic or something in a way it can support all kind of formats